src/plugins/crypto.c

```diff
@@ -154,6 +181,7 @@ gboolean bd_crypto_check_deps () {
  */
 gboolean bd_crypto_init () {
     c_locale = newlocale (LC_ALL_MASK, "C", c_locale);
+    crypt_set_log_callback (NULL, &crypto_log_redirect, NULL);
```

Comment:
I guess we also want to unset this if/when the plugin is unloaded, right?

Reply:
Fixed.